### DISCUSSION TITLE
linkerd Multi-cluster Is automatic failover supported?

### DISCUSSION THREAD

**Discussion by eddielishan** [linkerd/linkerd2#7927] [Q&A]
Excuse me，linkerd Multi-cluster Is automatic failover supported? For example, a service fails. Automatically transfer traffic to another cluster，If yes, how to configure or operate it. thank you!

**Comment by alpeb** [comment]
Hi @eddielishan . Nope, the multicluster extension limits itself to just mirroring services from external clusters, regardless of their availability.
